app/web/src/store/realtime/heimdall.ts

```diff
@@ -43,6 +43,7 @@ import {
   cachedAppEmitter,
   SHOW_CACHED_APP_NOTIFICATION_EVENT,
 } from "./cached_app_emitter";
+import { WorkspacePk } from "../workspaces.store";
```

Comment:
lets move this type out of the store, because we dont want to be importing stores.